operators/tokenizer/bert_tokenizer.cc

```diff
@@ -396,7 +396,7 @@ KernelHfBertTokenizer::KernelHfBertTokenizer(const OrtApi& api, const OrtKernelI
 void KernelHfBertTokenizer::Compute(const ortc::Tensor<std::string>& input,
                                     ortc::Tensor<int64_t>& output,
                                     ortc::Tensor<int64_t>& output1,
-                                    ortc::Tensor<int64_t>& output2,
+                                    std::optional<ortc::Tensor<int64_t>*> output2,
```

Comment:
nit: can we give the outputs meaningful names? `output`, `output1` and `output2` requires the reader to unnecessarily go through the implementation to figure out what each one is.

Reply:
added comments for the outputs here instead. hopefully that indicates the meaning more clearer. as we already have those named variables in the code below.